standards/code-style.adoc

```diff
@@ -619,16 +624,18 @@ different prefix for them.
 
 Example:
 
-[source, java]
+[source, kotlin]
 ----
-// Note the value of the field is the same as the name to avoid duplication issues
-static final String PREF_EMAIL = "PREF_EMAIL";
-static final String BUNDLE_AGE = "BUNDLE_AGE";
-static final String ARGUMENT_USER_ID = "ARGUMENT_USER_ID";
+companion object {
+    // Note the value of the field is the same as the name to avoid duplication issues
+    val PREF_EMAIL = "PREF_EMAIL"
```

Comment:
The ```const``` keyword can be added to these fields.

Reply:
Updated